src/dispatch/database/revisions/tenant/versions/2024-09-23_32652e0360dd.py

```diff
@@ -0,0 +1,105 @@
+"""empty message
```

Comment:
Could you add a quick description for this migration? It's not clear to me what it's about.

Reply:
Done.